pkg/pb/referralmanager.proto

```diff
@@ -23,16 +21,10 @@ message GetTokensResponse {
  repeated bytes token = 1;
 }
 
-message ReserveTokenRequest {
-    bytes token = 1;
-    bytes redeeming_satellite_id = 2 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
-}
-
-message ReserveTokenResponse{}
-
 message RedeemTokenRequest {
     bytes token = 1;
     bytes user_id = 2;
```

Comment:
I think it would make sense to change this to `redeem_user_id` and `redeem_satellite_id` just to eliminate any confusion on the referral manager side